src/js/providers/html5.js

```diff
@@ -483,6 +483,8 @@ define([
                 _this.supportsPlaybackRate = false;
             }
 
+            _setAttribute('preload', 'none');
```

Comment:
@robwalch Regarding your comment in the other PR

"That depends on when the source is set relative to when preload it. This is also one of the reasons why preload is an item/source property - so that they can be set at the same time or correct order, but now the player needs be in control of preload so the source's setting is completely irrelevant (so why are we still setting preload on source?)."